projects/1.20/assets/pedestals/pedestals/lang/zh_cn.json

```
@@ -0,0 +1,785 @@
+    "pedestals.upgrade_tooltip_blockcapacity_label": "额外操作的方块：",
+    "pedestals.upgrade_tooltip_itemcapacity_allowed": "可增加物品容量",
+    "pedestals.upgrade_tooltip_itemcapacity_label": "物品容量增加：",
+    "pedestals.upgrade_tooltip_fluidcapacity_allowed": "可增加流体容量",
+    "pedestals.upgrade_tooltip_fluidcapacity_label": "流体容量增加：",
+    "pedestals.upgrade_tooltip_energycapacity_allowed": "可增加能量缓存",
+    "pedestals.upgrade_tooltip_energycapacity_label": "能量缓存增加：",
+    "pedestals.upgrade_tooltip_xpcapacity_allowed": "可增加经验缓存量",
+    "pedestals.upgrade_tooltip_xpcapacity_label": "经验容量增加：",
+    "pedestals.upgrade_tooltip_dustcapacity_allowed": "可增加粉末存量",
+    "pedestals.upgrade_tooltip_dustcapacity_label": "粉末容量增加：",
+
+
+    "pedestals.upgrade_tooltip_area_allowed": "可提升工作范围/选区大小",
+    "pedestals.upgrade_tooltip_area_label": "水平范围提升：",
+
+    "pedestals.upgrade_tooltip_range_allowed": "可提升基座连接距离",
+    "pedestals.upgrade_tooltip_range_label": "连接距离提升：",
+
+    "pedestals.upgrade_tooltip_magnet_allowed": "可附加磁引效果",
+    "pedestals.upgrade_tooltip_magnet_label": "拥有磁引效果",
+
+    "pedestals.upgrade_tooltip_gentle_allowed": "可附加自动补种",
+    "pedestals.upgrade_tooltip_gentle_label": "拥有自动补种",
+
+    "pedestals.upgrade_tooltip_superspeed_allowed": "可附加超级加速",
+    "pedestals.upgrade_tooltip_superspeed_label": "已附加超级加速",
+
+    "pedestals.upgrade_tooltip_operatetobedrock_allowed": "可安装基岩钻井升级",
+    "pedestals.upgrade_tooltip_operatetobedrock_label": "已安装基岩钻井升级",
+
+    "pedestals.upgrade_tooltip_entitycontainer_allowed": "可以安装实体物品栏连接升级",
+    "pedestals.upgrade_tooltip_entitycontainer_label": "已安装实体物品栏连接升级",
+
+    "pedestals.upgrade_tooltip_remotestorage_allowed": "可以安装远程存储连接升级",
+    "pedestals.upgrade_tooltip_remotestorage_label": "已安装远程存储连接升级",
+
+
+    "pedestals.error": "错误",
+    "pedestals.description_shift": "按“Shift”键查看更多信息",
+    "pedestals.description_alt": "按“Alt”键查看更多信息",
+
+    "pedestals.mode_changed": "模式变更：",
+    "pedestals.mode_items": "物品",
+    "pedestals.mode_fluids": "流体",
+    "pedestals.mode_energy": "能量",
+    "pedestals.mode_experience": "经验",
+    "pedestals.mode_dust": "粉末",
+
+    "pedestals.mode_augment_particle": "禁用：粒子效果",
+    "pedestals.mode_augment_item": "禁用：物品模型渲染",
+    "pedestals.mode_augment_upgrade": "禁用：已安装升级渲染",
+    "pedestals.mode_augment_pi": "禁用：粒子效果/物品模型渲染",
+    "pedestals.mode_augment_pu": "禁用：粒子效果/已安装升级渲染",
+    "pedestals.mode_augment_iu": "禁用：物品模型/已安装升级渲染",
+    "pedestals.mode_augment_piu": "禁用：粒子效果/物品模型/已安装升级渲染",
+
+    "pedestals.tool_link_success_linkingtool": "连接成功，正在向接收端发送",
+    "pedestals.tool_link_success_backwardslinkingtool": "连接成功，正在从发送端接收",
+    "pedestals.tool_link_unsucess": "连接失败，已达到最大连接数量",
+    "pedestals.tool_link_removed": "连接已移除",
+    "pedestals.tool_link_itsself": "无法连接到它自身",
+    "pedestals.tool_link_network": "无法连接到另一个网络",
+    "pedestals.tool_link_distance": "连接失败，超出范围",
+    "pedestals.tool_link_start_linkingtool": "接收端基座坐标已记录。",

```

```suggestion
    "pedestals.tool_link_start_linkingtool": "已记录接收端基座坐标。",
```
具体以游戏内哪个读着更舒服为准
一般来说主谓宾结构更好一些